packages/table/src/react/__stories__/index.story.tsx

```diff
@@ -435,7 +435,7 @@ storiesOf('drawer', module)
     return (
       <Table>
         <Drawer base={baseRow} startOpen>
-          <Table inDrawer>
```

Comment:
we should update the docs if this isn't a thing anymore

https://github.com/pluralsight/design-system/blob/0b544456b4325e5d5c509afde0197fa4e8142508/docs/src/content/components/table.mdx